elementsd/Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM debian:buster-slim as builder
+
+RUN apt-get update -y \
+    && apt-get install -y \
+        automake \
+        autotools-dev \
+        bsdmainutils \
+        build-essential \
+        git \
+        libboost-chrono-dev \
+        libboost-filesystem-dev \
+        libboost-system-dev \
+        libboost-test-dev \
+        libboost-thread-dev \
+        libevent-dev \
+        libminiupnpc-dev \
+        libssl-dev \
+        libtool \
+        libzmq3-dev \
+        pkg-config \
+        python3 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+ENV ELEMENTS_VERSION=0.18.1.7
+ENV ELEMENTS_SRC=/usr/local/src/elements
+ENV ELEMENTS_BIN=/elements
+
+RUN git clone -n https://github.com/ElementsProject/elements $ELEMENTS_SRC \
+    && cd $ELEMENTS_SRC \
+    && git checkout tags/elements-$ELEMENTS_VERSION
+
+WORKDIR $ELEMENTS_SRC
+
+RUN ./autogen.sh
+
+RUN mkdir $ELEMENTS_BIN
+
+RUN ./configure --disable-tests --disable-bench --disable-wallet --without-gui --prefix=$ELEMENTS_BIN
+
+RUN V=1 make clean
+
+RUN V=1 make -j4
+
+RUN make install
+
+
+
+
+FROM debian:buster-slim
+
+LABEL maintainer.0="Dr Maxim Orlovsky <orlovsky@pandoracore.com>"
+LABEL maintainer.0="Alekos Filini"
```

Comment:
```suggestion
LABEL maintainer.0="Dr Maxim Orlovsky <orlovsky@pandoracore.com>" \
      maintainer.1="Alekos Filini"
```

Reply:
Found in guides: labels are just key-values; if keys are not unique they are replacing each other; and each LABEL command adds image build step, so better to use it once